x/executionlayer/types/genesis.go

```diff
@@ -51,18 +54,28 @@ type WasmCosts struct {
 	OpcodesDivisor    uint32 `json:"opcodes_divisor" toml:"opcodes-divisor"`
 }
 
+type DeployConfig struct {
+	MaxTtlMillis    uint32 `json:"max-ttl-millis" toml:"max-ttl-millis"`
+	MaxDependencies uint32 `json:"max-dependencies" toml:"max-dependencies"`
+}
+
+const (
+	mintCodePath = "$HOME/.nodef/contracts/mint_install.wasm"
+	posCodePath  = "$HOME/.nodef/contracts/pos_install.wasm"
+)
+
 // NewGenesisState creates a new genesis state.
 func NewGenesisState(genesisConf GenesisConf, accounts []Account, chainName string) GenesisState {
-	return GenesisState{GenesisConf: genesisConf, Accounts: accounts, ChainName: chainName,}
+	return GenesisState{GenesisConf: genesisConf, Accounts: accounts, ChainName: chainName}
 }
 
 // DefaultGenesisState returns a default genesis state
 func DefaultGenesisState() GenesisState {
 	genesisConf := GenesisConf{
 		Genesis: Genesis{
 			Timestamp:       0,
-			MintWasm:        DefaultMintWasm,
-			PosWasm:         DefaultPosWasm,
+			MintWasm:        util.LoadWasmFile(os.ExpandEnv(mintCodePath)),
```

Comment:
If we load genesis system contract as file, we cannot assure every user's DefaultGenesis has the same contract bytes included by genesis.json. Also if user wants to use another own system contract, user can load through load-chainspec command.

Reply:
`nodef load-chainspec \`
`          --pos-wasm-path blahblah.wasm`
`          --mint-wasm-path blahblah.wasm`
Looking forward as like this?

Reply:
@psy2848048  When doing load-chainspec, the system-contract path is already included. The problem is integration testing.